api/src/main/java/ai/djl/translate/Translator.java

```diff
@@ -75,14 +72,16 @@
      *
      * @return the {@link Batchifier}
      */
-    Batchifier getBatchifier();
+    default Batchifier getBatchifier() {
+        return null;
```

Comment:
I'm not sure I agree with this default because it makes it confusing whether to translate Record -> Record or Batch -> Batch. So far, we have mainly focused on Record -> Record while treating the batch one as a special case while this default changes that. This would be especially confusing for training users.

What if we make the default translator use StackBatchifier for training. Then, add a new interface like `BatchTranslator extends Translator` with a `null` default that can explain how it affects translation and the pre/post-processing in comparison to regular usages of the translator.